docs/configuration.rst

```diff
@@ -153,33 +153,16 @@ These settings are global and have effect on both, tooltips and modal dialogues.
 .. _Mathjax: http://www.sphinx-doc.org/es/master/usage/extensions/math.html#module-sphinx.ext.mathjax
 
 
-.. warning::
-
-   You shouldn't modify the following three settings (api_host, project, version) unless you know what you are doing.
-   Their defaults should be fine to build the documentation and make it work in Read the Docs.
-
-
 .. confval:: hoverxref_api_host
 
    Description: Host URL for the API to retrieve the content of the floating window
 
-   Default: ``https://readthedocs.org``
-
-   Type: string
-
-.. confval:: hoverxref_project
-
-   Description: Read the Docs project slug
-
-   Default: It defaults to ``READTHEDOCS_PROJECT`` environment variable
-
-   Type: string
-
-.. confval:: hoverxref_version
+   .. warning::
 
-   Description: Read the Docs version slug
+     You shouldn't modify this setting unless you know what you are doing.
+     Its default should be fine to build the documentation and make it work in Read the Docs.
 
-   Default: It defaults to ``READTHEDOCS_VERSION`` environment variable
+   Default: ``https://readthedocs.org``
```

Comment:
How are we planning to support this on .org & .com? Should we do this automagically, or just always default to .org?

Reply:
Hrm... _good question_!

I haven't thought about this, honestly. I don't think we have a way to differentiate if we are running in .org or .com from the builder. It may be good to expose an environment variable? In that case, we can use it to differentiate.

Actually... I just remembered that @stsewd already mentioned this in https://github.com/readthedocs/sphinx-hoverxref/pull/134/ --so, we should probably continue the conversation there, finish that PR, and merge it. I think it was good, but there was an issue with CORS that I don't remember where we ended up.